sops/howto/derive_ocm_cluster_id_from_connector_cluster_id.asciidoc

```diff
@@ -0,0 +1,51 @@
+// begin header
+ifdef::env-github[]
+:tip-caption: :bulb:
+:note-caption: :information_source:
+:important-caption: :heavy_exclamation_mark:
+:caution-caption: :fire:
+:warning-caption: :warning:
+endif::[]
+:numbered:
+:toc: macro
+:toc-title: pass:[<b>Table of Contents</b>]
+
+// end header
+= Derive OCM Cluster ID from Connector Cluster ID
+
+toc::[]
+
+== Description
+
+The purpose of this SOP is to describe the process of deriving the OCM cluster id using the connector cluster id. This is useful when you need the retrieve the OCM cluster id for troubleshooting any issue/alert.
+
+== Prerequisites
+
+1. https://github.com/bf2fc6cc711aee1a0c2a/cos-tools/releases[RHOC cli]
+2. https://github.com/openshift-online/ocm-cli[OCM cli]
+3. Retrieve your https://console.redhat.com/openshift/token[ocm-offline-token] using your Red Hat account
+- Follow the steps on that page to run the ocm login with the provided token
+
+== Derive OCM cluster ID from Connector Cluster ID
+
+The following command prints the OCM cluster id
+----
+rhoc cc get --id ${cluster-id} --ocm | jq -r '.status.platform.id'
```

Comment:
This derives the external id, not the OCM ID.
The OCM ID is in `ocm.id`

However, since it turn out the external id can be used for backplane, I wonder if the  external id is enough 

@lgarciaaco @pat-cremin what do you think ?

Reply:
External ID or ocm ID will be sufficient for SRE needs, so either way works for us.

Reply:
@lburgazzoli, Do we need any change here?

Reply:
```suggestion
rhoc cc get --id ${cluster-id} | jq -r '.status.platform.id'
```